finance_research/pipeline.py

```
@@ -0,0 +1,161 @@
+                title=f"{company_name} 财报分析报告",
+                companies=str([company_name]),
+                content=report_md,
+                citations=str(citations),
+            )
+        )
+        return {"report": report_md, "citations": citations, "analysis": analysis}
+
+    def compare(self, companies: List[str], queries: List[str]) -> dict:
+        """Discover and compare multiple companies."""
+        per_company = {}
+        for company, query in zip(companies, queries):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)